mlflow/tracing/types/constant.py

```diff
@@ -1,11 +1,14 @@
 # NB: These keys are placeholders and subject to change
 class TraceMetadataKey:
-    NAME = "name"
     INPUTS = "inputs"
     OUTPUTS = "outputs"
```

Comment:
While we're here, can we rename `TraceMetadataKey.INPUTS` and `TraceMetadataKey.OUTPUTS` to `mlflow.traceInputs` and `mlflow.traceOutputs`?

Reply:
Yup!